x/concentrated-liquidity/pool.go

```diff
@@ -209,11 +230,28 @@ func (p Pool) CalcInAmtGivenOut(ctx sdk.Context, tokensOut sdk.Coins, tokenInDen
 		priceNext := priceNextTop.Quo(priceNextBot)
 
 		// new amount in, will be needed later
-		// amountIn = calcAmount1(liq, priceNext, sqrtPCurTick)
+		// amountOut = calcAmount1(liq, priceNext, sqrtPCurTick)
 		amountIn := calcAmount1(liq, priceNext, sqrtPCurTick)
 		// fee logic
 		amountIn = amountIn.Quo(sdk.OneDec().Sub(swapFee))
 		return sdk.NewCoin(tokenInDenom, amountIn.TruncateInt()), nil
 	}
 	return sdk.Coin{}, fmt.Errorf("tokenIn does not match any asset in pool")
 }
+
+func (p *Pool) SetInitialPoolAssets(poolDenoms []string) error {
```

Comment:
Does it make sense to call this just denoms instead since pool assets in gamm contain tokens and weights? My concern is confusion with the gamm's pool asset concept

Reply:
Agreed, will change